File: backend/services/license-service/src/main/java/com/lmspilot/license/LicenseServiceApplication.java
package com.lmspilot.license;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class LicenseServiceApplication {
    public static void main(String[] args) { SpringApplication.run(LicenseServiceApplication.class, args); }
}

File: backend/services/license-service/src/main/kotlin/com/lmspilot/license/LicenseServiceApplication.kt
package com.lmspilot.license

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class LicenseServiceApplication

fun main(args: Array<String>) {
    runApplication<LicenseServiceApplication>(*args)
}
